src/Microsoft.DotNet.CodeFormatting.Tests/FormattingEngineCreationTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+using CodeFormatter;
+using Microsoft.DotNet.CodeFormatter.Analyzers;
+using Xunit;
+
+namespace Microsoft.DotNet.CodeFormatting.Tests
+{
+    public class FormattingEngineCreationTests
+    {
+
+        private static Assembly[] DefaultCompositionAssemblies =
+                                        new Assembly[] {
+                                            typeof(FormattingEngine).Assembly,
+                                            typeof(OptimizeNamespaceImportsAnalyzer).Assembly
+                                        };
+        // TODO: fix hardcoded path
+        private static string TestDLLDir = @"E:\src\codeformatter\src\Microsoft.DotNet.CodeFormatting.Tests\TestAnalyzers\";
```

Comment:
Yep. Should've put 'Ctrl-F for TODO in my list of TODOs' ;)

Been sidetracked with other work the past week or so but will be revisiting this and cleaning it up with any other CR comments in the next day or two, thanks for looking.